playbook.yml

```diff
@@ -0,0 +1,73 @@
+- name: Run todo app
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_key
+    prompt: What is your trello key?
+    private: no
+
+  - name: trello_token
+    prompt: What is your trello token?
+    private: no 

```

Comment:
Although this is fine for now, it's risking that someone later uses this file in a different way and doesn't spot that these should be sensitive - for example if they next run it in a pipeline and the secrets end up in the logs, so setting these to "yes" feels like a sensible defensive stance